### DISCUSSION TITLE
Unity WebGL Support

### DISCUSSION THREAD

**Discussion by DreamerChaserHAH** [omansak/libvideo#251] [Q&A]
This library does not work properly for Unity WebGL, as it does not support functions from specific classes like System.Threading, etc. The error was that unity webgl couldn't find "Libc.dll" at run time. It works properly in editor and standalone build as expected. Is there any way to make it compatible with WebGL builds?

**Comment by imdadmian** [comment]
Any Solution for this thread.?

**Comment by etienne-cowley** [comment]
Same here, any solution?

**Comment by DreamerChaserHAH** [comment]
Unfortunately, I couldn't find an answer just yet
